new/cookbook-abandonment.md

```diff
@@ -0,0 +1,50 @@
+---
+RFC: unassigned
+Author: Nell Shamrell-Harrington <nshamrell@chef.com>
+Status: Draft
+Type: Process
+---
+
+# Cookbook Abandonment On Supermarket
+
+The Chef community needs a process around handling when cookbooks on Supermarket are abandoned.  This occurs when a cookbook owner does not respond to requests to adopt the cookbook, and the cookbook has not been updated in some time.
```

Comment:
This would read much easier if it was "The Chef community needs a process for handling cookbooks on Supermarket that have become abandoned."